build/common.sh

```diff
@@ -1246,14 +1246,13 @@ function kube::release::gcs::verify_ci_ge() {
       greater=false
     elif [[ "${version_patch}" -gt "${gcs_version_patch}" ]]; then
       : # fall out
-    elif [[ "${version_prerelease}" =~ alpha.* && "${gcs_version_prerelease}" == "beta" ]]; then
+    elif [[ "${version_prerelease}" < "${gcs_version_prerelease}" ]]; then
```

Comment:
Correct.  This phrase is lexicographically comparing strings, not integers.  `-lt` is for ints, `<` is for strings.